MONGOID-4998 deprecate :id_sort and implement limit positional argument on #first/#last (#5358)
mongodb/mongoid: docs/release-notes/mongoid-7.5.txt

option has been deprecated in Mongoid 7.5 and it is recommended to use ``#take``
instead going forward. Support for the ``:id_sort`` option will be dropped in
Mongoid 8.

Deprecate ``:id_sort`` Option and Support ``limit`` on ``#first/last``
----------------------------------------------------------------------

> Mongoid 7.5 deprecates support for the ``:id_sort`` keyword argument for the

Reviewer: ```suggestion
Mongoid 7.5 deprecates the ``:id_sort`` keyword argument for the
```

> ``Criteria#first`` and ``Criteria#last`` methods. These methods have added

Reviewer: ```suggestion
``Criteria#first`` and ``Criteria#last`` methods. Please use ``Criteria#take`` to retrieve documents without sorting by id. ```

> support for adding a limit as a positional argument:

Reviewer: ```suggestion
The ``first`` and ``last`` methods now take the number of documents to return as a positional argument, mirroring the functionality of Ruby's ``Enumerable`` method and ActiveRecord's ``first`` and ``last`` methods. Both invocations (with limit as positional arguments and with ``:id_sort`` option) remain supported in Mongoid 7.x, but the ``:id_sort`` invocation will be removed in Mongoid 8.
```
